ceph/ceph.py

```diff
@@ -1297,6 +1293,19 @@ def __init__(
     def client(self):
         return self.get_client()
 
+    def _get_ssh_key(self, private_key_file_path):
+        """Get SSH key based on file type"""
+        with open(private_key_file_path, "r") as f:
+            _header = f.readline().strip()
+
+        if "BEGIN OPENSSH PRIVATE KEY" in _header:
```

Comment:
An `ecdsa` key also can begin with this line.

Reply:
@psathyan ecdsa private key starts with `BEGIN EC PRIVATE KEY`. Refer -
1. https://crypto.stackexchange.com/questions/48707/ecdsa-private-key-format
2. https://techdocs.akamai.com/iot-token-access-control/docs/generate-ecdsa-keys

Reply:
I believe that is when we use `openssl` however, we generate using `ssh-keygen`

Reply:
Can u plz suggest alternative steps to differentiate keys ?